src/helpers/MdObjectHelper.ts

```diff
@@ -109,17 +111,21 @@ export const mdObjectToGeoPushpinBucketProps = (
         afmFilter => !AFM.isExpressionFilter(afmFilter),
     ) as VisualizationObject.VisualizationObjectExtendedFilter[];
 
-    const configWithTooltipText: IGeoConfig = {
-        ...config,
-        tooltipText,
-    };
+    const { tooltipText: tooltipTextDisplayForm } = config;
+    const configWithTooltipText: IGeoConfig =
+        tooltipTextDisplayForm && typeof tooltipTextDisplayForm === "string" // tooltipText uri was built from properties.control
+            ? {
+                  ...config,
+                  tooltipText: buildAttributeBucket(tooltipTextDisplayForm, TOOLTIP_TEXT),
+              }
+            : config;
```

Comment:
How about?
```
const configWithTooltipText: IGeoConfig = { ...config };
if (tooltipTextDisplayForm && typeof tooltipTextDisplayForm === "string") {
     configWithTooltipText.tooltipText = buildAttributeBucket(tooltipTextDisplayForm, TOOLTIP_TEXT);
}
```

Reply:
Yes, I agree and want to add some
1. `configWithTooltipText.tooltipText =  attribute(uri).localIdentifier(localIdentifier)` so we don't have to create new 
single-use `buildAttributeBucket `
2. long `tooltipTextDisplayForm && typeof tooltipTextDisplayForm === "string"` should be assigned to meaningful const. And what other types can `tooltipTextDisplayForm ` hold besides `string`?

Reply:
fixed